deploy/rbac.yaml

```diff
@@ -4,6 +4,11 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: keycloak-operator
 rules:
+  - apiGroups:
+      - batch
+    resources:
+      - "cronjobs"
+    verbs: [get, list, create, update, delete]
```

Comment:
`deletecollection` is missing here. Without it deprovision won't work.

Reply:
I was trying to be too clever here, pushed a correction now.